build.cmd

```diff
@@ -1,5 +1,7 @@
+set build_type=%1
 mkdir build
 cd build
-cmake ../
-MSBuild revengeMusic.sln
-cd ..
+mkdir %build_type%
```

Comment:
Also it would be nice to add return to the directory in which the command was run at.